django/contrib/contenttypes/migrations/0002_removed_content_type_name.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
I've named the other migration files in present tense ("remove_content_type...")